index.html

```diff
@@ -2599,6 +2608,124 @@ <h3>
+        <!-- todo: return value -->
+        <pre class="idl">
+          partial interface Navigator {
+              [SecureContext, NewObject] Promise&lt;undefined&gt; install();
+          };
+        </pre>
+        <h3>
+          Internal Slots
+        </h3>
+        <p>
+          This API adds the following internal slot to the {{Navigator}}
+          interface.
+        </p>
+        <dl>
+          <dt>
+            {{Promise}}? <dfn>[[\installPromise]]</dfn>
+          </dt>
+          <dd>
+            The [=this=].{{Navigator/[[installPromise]]}} is a promise that
+            represents a user's current intent to install an application. It is
+            initialized to `null`.
+          </dd>
+        </dl>
+        <h3>
+          <dfn>install()</dfn> method
+        </h3>
+        <p>
+          When the {{Navigator/install()}} method is called, run the following
+          steps to [=install=] the website.
+        </p>
+        <ol class="algorithm">
+          <li>Let |global:Window| be [=this=]'s [=relevant global object=].
+          </li>
+          <li>Let |document:Document| be |global|'s [=associated Document=].
+          </li>
+          <li>If |document| is not [=Document/fully active=], return [=a
+          promise rejected with=] an {{"InvalidStateError"}} {{DOMException}}.
+          </li><!-- todo: permission policy? -->
+          <li>If [=this=].{{Navigator/[[installPromise]]}} is not `null`,
+          return [=a promise rejected with=] an {{"InvalidStateError"}}
+          {{DOMException}}.
+          </li>
+          <li>If |global| does not have [=transient activation=], return [=a
+          promise rejected with=] a {{"NotAllowedError"}} {{DOMException}}.
+          </li>
+          <li>[=Consume user activation=] of |global|.
+          </li>
+          <li>Set [=this=].{{Navigator/[[installPromise]]}} to be <a>a new
+          promise</a>.
+          </li>
+          <li>Return [=this=].{{Navigator/[[installPromise]]}} and <a>in
+          parallel</a>:
+            <ol>
+              <li>[=Processing|Process=] the |document|'s [=manifest=].
+              </li>
+              <li>If the |document|'s [=Document/processed manifest=] is null,
+              [=queue a global task=] on the [=user interaction task source=]
+              using |global| to:
+                <ol>
+                  <li>[=Reject=] [=this=].{{Navigator/[[installPromise]]}} with
+                  an {{"DataError"}} {{DOMException}}.
+                  </li>
+                  <li>Set [=this=].{{Navigator/[[installPromise]]}} to `null`.
+                  </li>
+                  <li>Terminate this algorithm.
+                  </li>
+                </ol>
+              </li>
+              <li>Present an installation UI for the |document|'s
+              [=Document/processed manifest=].
+                <aside class="note">
+                  If a user agent only allows one instance of the application
```

Comment:
There's a lot of RFC2119 language here, which is not appropriate for a note.

Reply:
Removed the note.